composer.json

```diff
@@ -64,6 +64,10 @@
             "loader-dev": {
                 "phel\\": "tests/phel/"
             },
+            "export" : {
+                "target-directory" : "src/PhelGenerated",
+                "prefix-namespace" : "PhelGenerated"
+            },
```

Comment:
What do you think about this, @jenshaase? We should add these config options in the getting started guide (+ scaffolding)

Reply:
Can we rename `prefix-namespace` to `namespace-prefix`.

However, we don't needs these config options in here. They are just there so you can test?

Regarding documentation: It should not be added to getting started. That too much. It should be added to "PHP Interop" and "Configuration".

Reply:
> Can we rename prefix-namespace to namespace-prefix.

Done

> However, we don't need these config options here. They are just there so you can test?

Yes, I leave them there so you can actually easily test them. It doesn't affect the real usage of the end-user.

> Regarding documentation: It should not be added to getting started. That too much. It should be added to "PHP Interop" and "Configuration".

Done.